openedx/core/djangoapps/programs/migrations/0005_programsapiconfig_max_retries.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('programs', '0004_programsapiconfig_enable_certification'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='programsapiconfig',
+            name='max_retries',
+            field=models.PositiveIntegerField(default=3, help_text='When making requests to the Programs API, make at most this many attempts to retry a failing request.', verbose_name='Maximum API Retries'),
```

Comment:
You need to recreate this migration. The `help_text` here doesn't match the actual text in models, which will confuse Django's migration framework.